docs/getting-started-guides/fedora/fedora_ansible_config.md

```diff
@@ -58,7 +59,7 @@ We will set the kube_ip_addr to '10.254.0.[1-3]', for now.  The reason we do thi
 [etcd]
 192.168.121.205
 
-[minions]
+[nodes]
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  tapiohush@gmail.com
    Domain eikaas.no has exceeded the max emails per hour (33/30 (110%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o4.sgmail.github.com ([192.254.112.99]:47924)
	by cpanel30.proisp.no with esmtps (TLSv1.2:DHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.85)
	(envelope-from <bounces+848413-7097-robin=eikaas.no@sgmail.github.com>)
	id 1ZEhbA-0020m7-6y
	for robin@eikaas.no; Mon, 13 Jul 2015 19:33:05 +0200
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=cAsuvnVBmKWvGg0TQ37dT4937gE=; b=FyZosbImUBzz6Y6H
	4SPF/OHYAsRhWbtRZge7A35Qy0gBOHvG4LwSiirDwSA2N/eFKGKr1pwX3DJUxF05
	dCE6N/OooP23zw+qpLcMGMgi1tysbVcXYs7UMy4+QeJoI74+uBb+h9f1w8Bj/mDL
	JawTskkvxNoR11zgJGW7NSus6+o=
Received: by filter0472p1mdw1.sendgrid.net with SMTP id filter0472p1mdw1.16583.55A3F64616
        2015-07-13 17:32:56.478957164 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd-004 (SG) with ESMTP id 14e887a0b4c.3ac.4a7c8
	for <robin@eikaas.no>; Mon, 13 Jul 2015 17:32:56 +0000 (UTC)
Date: Mon, 13 Jul 2015 10:32:56 -0700
From: Brian Grant <notifications@github.com>
Reply-To: GoogleCloudPlatform/kubernetes <reply@reply.github.com>
To: GoogleCloudPlatform/kubernetes <kubernetes@noreply.github.com>
Message-ID: <GoogleCloudPlatform/kubernetes/pull/11130/r34488430@github.com>
In-Reply-To: <GoogleCloudPlatform/kubernetes/pull/11130@github.com>
References: <GoogleCloudPlatform/kubernetes/pull/11130@github.com>
Subject: Re: [kubernetes] Change 'minion' to 'node' in docs (#11130)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_55a3f648606bb_6fde3f88cf11b2b81043610";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: bgrant0607
X-GitHub-Recipient: eikaas
List-ID: GoogleCloudPlatform/kubernetes
 <kubernetes.GoogleCloudPlatform.github.com>
List-Archive: https://github.com/GoogleCloudPlatform/kubernetes
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+005f5f67d4d0ce18d84c4e74391ddadeb463aa202ca7bdfe92cf0000000111bbb84892a170ce020e406e@reply.github.com>,
 <https://github.com/notifications/unsubscribe/AF9fZ83D6uDxKOP5V8KZIc_8aEhJwAHsks5oc-3IgaJpZM4FXLAW>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: robin@eikaas.no
X-SG-EID: D8AxvQssagt16ZlBezT4ZE8ej34nUdhd0RF2VqeF0hxLCBqZiBp1ZeAyeUGiyNqoWFqUELYTX0/7Jv
 78I8KmCEC6zxixNeNpa6vvg0PQ5LHgbQ+gojd/hrPSOaRwSfXiB3OW98YAQcH5ETtsb4fDJWsrS23f
 mJ3HgM0gVYPyE+U=
X-Spam-Status: No, score=-1.3
X-Spam-Score: -12
X-Spam-Bar: -
X-Ham-Report: Spam detection software, running on the system "cpanel30.proisp.no",
 has NOT identified this incoming email as spam.  The original
 message has been attached to this so you can view it or label
 similar future email.  If you have any questions, see
 root\@localhost for details.
 
 Content preview:  > @@ -58,7 +59,7 @@ We will set the kube_ip_addr to '10.254.0.[1-3]',
    for now. The reason we do thi > [etcd] > 192.168.121.205 > > -[minions] >
    +[nodes] Yeah, I doubt this can be changed without also updating https://github.com/eparis/kubernetes-ansible
    [...] 
 
 Content analysis details:   (-1.3 points, 4.0 required)
 
  pts rule name              description
 ---- ---------------------- --------------------------------------------------
  0.0 URIBL_BLOCKED          ADMINISTRATOR NOTICE: The query to URIBL was blocked.
                             See
                             http://wiki.apache.org/spamassassin/DnsBlocklists#dnsbl-block
                              for more information.
                             [URIs: github.com]
 -0.0 RCVD_IN_MSPIKE_H4      RBL: Very Good reputation (+4)
                             [192.254.112.99 listed in wl.mailspike.net]
 -0.3 RP_MATCHES_RCVD        Envelope sender domain matches handover relay domain
 -1.9 BAYES_00               BODY: Bayes spam probability is 0 to 1%
                             [score: 0.0000]
  0.0 HTML_MESSAGE           BODY: HTML included in message
  1.1 HTML_IMAGE_ONLY_16     BODY: HTML: images with 1200-1600 bytes of words
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from author's
                             domain
 -0.1 DKIM_VALID             Message has at least one valid DKIM or DK signature
  0.1 DKIM_SIGNED            Message has a DKIM or DK signature, not necessarily valid
 -0.0 RCVD_IN_MSPIKE_WL      Mailspike good senders
X-Spam-Flag: NO

----==_mimepart_55a3f648606bb_6fde3f88cf11b2b81043610
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -58,7 +59,7 @@ We will set the kube_ip_addr to '10.254.0.[1-3]', for now.  The reason we do thi
>  [etcd]
>  192.168.121.205
>  
> -[minions]
> +[nodes]

Yeah, I doubt this can be changed without also updating https://github.com/eparis/kubernetes-ansible

---
Reply to this email directly or view it on GitHub:
https://github.com/GoogleCloudPlatform/kubernetes/pull/11130/files#r34488430
----==_mimepart_55a3f648606bb_6fde3f88cf11b2b81043610
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11130#discussion_r34488430">docs/getting-started-guides/fedora/fedora_ansible_config.md</a>:</p>
<pre style='color:#555'>&gt; @@ -58,7 +59,7 @@ We will set the kube_ip_addr to &#39;10.254.0.[1-3]&#39;, for now.  The reason we do thi
&gt;  [etcd]
&gt;  192.168.121.205
&gt;  
&gt; -[minions]
&gt; +[nodes]
</pre>
<p>Yeah, I doubt this can be changed without also updating <a href="https://github.com/eparis/kubernetes-ansible">https://github.com/eparis/kubernetes-ansible</a></p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11130/files#r34488430">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/AF9fZ9B9EvGTMl_XjCTDkUKbwlSqUDb2ks5oc-3IgaJpZM4FXLAW.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11130/files#r34488430"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_55a3f648606bb_6fde3f88cf11b2b81043610--